ansible/roles/etcd/tasks/start.yml

```diff
@@ -1,6 +1,2 @@
 ---
-- include: etcd-start.yml
-  when: not is_coreos
-
-- include: etcd2-start.yml
-  when: is_coreos
+- include: "{% if not is_coreos %}etcd-start{% else %}etcd2-start{% endif %}.yml"
```

Comment:
I think it should take a little time to do this and testing ...
Should i just refactoring these include statement of the 'etcd' role this pr?